Fix/IndicationBook.cs

```diff
@@ -0,0 +1,290 @@
+using System;
+using System.Linq;
+using static Fix.Dictionary;
+
+namespace Fix
+{
+    public class IndicationsEventArgs : EventArgs
+    {
+        public IndicationsEventArgs(Indication indication)
+        {
+            Indication = indication;
+        }
+
+        public Indication Indication { get; }
+    }
+
+    public class IndicationBook
+    {
+        #region Events
+
+        public delegate void IndicationDeligate(object sender, IndicationsEventArgs e);
+
+        public event IndicationDeligate? IndicationInserted;

```

Comment:
These events can use System.EventHandler<Indication>, see https://docs.microsoft.com/en-us/dotnet/api/system.eventhandler-1?view=net-5.0

I realise this is copied code, I'll look at fixing other similar cases.

Reply:
Followed your pattern.